### DISCUSSION TITLE
CNCF TAG-Runtime update/presentation

### DISCUSSION THREAD

**Discussion by raravena80** [cri-o/cri-o#6433] [Ideas]
Hello cri-o team,

I'm one of the co-chairs of the CNCF TAG-Runtime. I'm reaching out and I think it would be great for you to present/discuss an update/roadmap of the project at one of our meetings. For example, we are interested in how the project would interact with WebAssembly modules in the future.

If this sounds good, feel free to add it to our [agenda](https://bit.ly/cncf-tag-runtime-meeting-notes) or reach out to me (raravena80 at gmail.com)

Thanks!

**Comment by haircommander** [comment]
I like this idea! Given the holidays are approaching, maybe we can aim for January 19th meeting?

**Comment by raravena80** [comment]
Sounds good to me!
